packages/store/src/decorators/select/select.ts

```diff
@@ -11,7 +12,7 @@ export function Select<T>(rawSelector?: T, ...paths: string[]): PropertyDecorato
   return function(target, key): void {
     const name: string = key.toString();
     const selectorId = `__${name}__selector`;
-    const selector = createSelectorFn(name, rawSelector, paths);
+    let selector: SelectorFn | null = null;
```

Comment:
Could there be any unintended consequence of this only being don in the `get`?
I can't think of any, but it is worth a thought.